src/renderer/sentry/initialize.injectable.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import initializeSentryReportingWithInjectable from "../../common/error-reporting/initialize-sentry-reporting.injectable";
+import setupAppPathsInjectable from "../app-paths/setup-app-paths.injectable";
+import { beforeFrameStartsInjectionToken } from "../before-frame-starts/before-frame-starts-injection-token";
+import { init } from "@sentry/electron/renderer";
+
+const initializeSentryReportingInjectable = getInjectable({
+  id: "initialize-sentry-reporting",
+  instantiate: (di) => ({
+    id: "initialize-sentry-reporting",
+    run: () => {
+      // Have to inject this here instead of above so that its dependency on `setupAppPathsInjectable` doesn't throw
+      const initializeSentryReportingWith = di.inject(initializeSentryReportingWithInjectable);
+
+      if (process.isMainFrame) {
```

Comment:
Side-effect, and intent of a technicality is unclear. We do have `currently-in-cluster-frame.injectable.ts` which could be the single source of truth for handling `process.isMainFrame`. Be cautious that currently it either doesn't advertise causing side-effects.

Reply:
Already changed in future commit.